### PR TITLE
Solved MEMBERSHIP 

### LINKED ISSUE #686: MEMBERSHIP heading text isn't responsive
### Description

The word "MEMBERSHIP" in the
`<div class="col-sm-12"><span class="price">Membership</span></div>`

overlaps the text paragraph next to it for the screen sizes from 976px to 1184px.

### Screenshot
Screenshot (34)

Look into the similar issues #390  and #627 .

### PR DESCRIPTION
**Pull request by tikhsuP**
Fix #686

### CODE REVIEW

**Review by Chaitanyassr — CHANGES_REQUESTED** [review]
Please provide the deployed link.
also changing the respective size of the membership would be better .

**Comment by tikhsuP** [comment]
@Chaitanyassr 
The updated section looks like this:

Capturefossasiaaa

**Comment by Chaitanyassr** [comment]
> @Chaitanyassr
> The updated section looks like this:
> 
> Capturefossasiaaa

Can you please provide a deployed link? either you can deploy on netlify or you can go the enviromnent section of you GitHub repo and paste the link here.

**Comment by tikhsuP** [comment]
@Chaitanyassr 
The deployed link: [https://tikhsup.github.io/fossasia.org/](https://tikhsup.github.io/fossasia.org/)

**Review by Chaitanyassr — CHANGES_REQUESTED** [review]
why doing extra change?

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-fossasia.org
```
Comment (Chaitanyassr) [inline CNAME:1]: why deleting this?

Comment (tikhsuP) [reply CNAME:1]: I apologize for this, but GitHub was showing me a problem while deploying my branch due to same CNAME.
So, I temporarily deleted that file. If you are satisfied with the change made in the MEMBERSHIP text, then I can add the CNAME file again.

**Comment by tikhsuP** [comment]
@Chaitanyassr are you satisfied?

File: CNAME (hunk 1 above)
Comment (Chaitanyassr) [reply CNAME:1]: What kind of problem? Add the file as a PR should include minimum changes for a particular fix. For the deploymnet part, You can always use another method for deployment like netlify. Also commits need to be squashed . It should be one or two commit for one line change. thank you.

**Comment by tikhsuP** [comment]
@Chaitanyassr  Review.

File: CNAME (hunk 1 above)
Comment (abishekvashok) [reply CNAME:1]: Please don't remove cname, it's required for deployment via GitHub pages and custom domain

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Pull request closed without merging**